Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -51,10 +94,58 @@ public static int[] RemoveFirst(int[] array)
         /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveAt(int[] array, int index)
         {
-            // ToDo: implement.
+            var newArray = ProcessArray(array, index);
+            return newArray;
+        }
+
+        private static int[] ProcessArray(int[] array, int index)
```

Comment:
Process array is a bad name. I have no idea what it does.

Reply:
Is already deleted in my new code